uss-api/swagger.yaml

```diff
@@ -131,9 +135,7 @@ paths:
           schema:
             $ref: >-
               https://raw.githubusercontent.com/nasa/utm-apis/master/utm-domains/utm-domain-commons.yaml#/definitions/UTMRestResponse
-      security:
-        - ussapi_security:
-            - utm.nasa.gov_write.message
+
```

Comment:
Is there no GET on /utm_messages?

Reply:
no published GET on utm-messages.

However, we have implemented one for NUSS.  It's codegen'ed into gov.nasa.utm.ussn.testapi;